functions/dir_clean.pp

```diff
@@ -0,0 +1,21 @@
+# @summary Take a path and normalises it to its unix form.
+# Instead of having to deal with the different seperators between unix and windows this
+# function instead formates windows paths a equivalent unix like path.
+#
+# AFAIK this type of unix like path will work on windows.
+#
+# @param dir The path to clean
+# @return Stdlib::Absolutepath The cleaned path
```

Comment:
I don't think you need to include the type for strings to detect it if there is already one. Maybe good to generate the reference doc to check.
```suggestion
# @return The cleaned path
```

If it is needed, please match it to the actual return type.

Reply:
fixed